core/indexer/elasticsearch.go

```diff
@@ -729,7 +727,7 @@ func buildRewardTransaction(
 		Round:         rTx.Round,
 		Value:         rTx.Value.String(),
 		Receiver:      hex.EncodeToString(rTx.RcvAddr),
-		Sender:        shardIdStr,
+		Sender:        "meta",
```

Comment:
magic string :) we have a couple of strings defining meta, maybe use one of them

Reply:
done